xfel/merging/application/scale/experiment_scaler.py

```diff
@@ -142,53 +144,47 @@ def run(self, experiments, reflections):
 
     return new_experiments, new_reflections
 
-  def fit_experiment_to_reference(self, model_intensities, experiment_intensities, matching_indices):
+  def fit_experiment_to_reference(self,
+      model_intensities,
+      experiment_intensities,
+      matching_indices,
+      weights='unit'
+  ):
     'Scale the observed intensities to the reference, or model, using a linear least squares fit.'
      # Y = offset + slope * X, where Y is I_r and X is I_o
 
+    assert weights in ['unit', 'icalc']
```

Comment:
Is this assertion needed, considering "weights" is a phil choice?

Reply:
No, it's more of a comment disguised as an assertion. I wouldn't mind taking it out.